apps/meteor/tests/data/rooms.helper.js

```diff
@@ -54,14 +54,14 @@ function actionRoom({ action, type, roomId }) {
 	return new Promise((resolve) => {
 		request
 			.post(api(`${endpoints[type]}.${action}`))
-			.set(credentials)
+			.set(overrideCredentials)
 			.send({
 				roomId,
 			})
 			.end(resolve);
 	});
 }
 
-export const deleteRoom = ({ type, roomId }) => actionRoom({ action: 'delete', type, roomId });
+export const deleteRoom = ({ type, roomId, overrideCredentials = credentials }) => actionRoom({ action: 'delete', type, roomId, overrideCredentials });
 
 export const closeRoom = ({ type, roomId }) => actionRoom({ action: 'close', type, roomId });
```

Comment:
this seems to not be used anymore, as it would not even work since it is not providing any credentials.. should it be removed?